pkg/util/coverage/coverage.go

```diff
@@ -0,0 +1,83 @@
+// +build coverage
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package coverage provides tools for coverage-instrumented binaries to collect and
+// flush coverage information.
+package coverage
+
+import (
+	"flag"
+	"github.com/golang/glog"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"os"
+	"testing"
+	"time"
+)
+
+const flushInterval = 5 * time.Second
```

Comment:
Can we make this configurable?

Reply:
Do we have a reason to do so?

Given the downsides of trying to parse flags here, this would probably be yet another environment variable to look at.

Reply:
Well.  I haven't used this in the wild yet so I don't know if we have a reason.  I just assume we'll want to tweak this at some point.  Guess we can wait to see if a follow on PR is necessary

Reply:
discussed this offline, seems easier to go ahead and add something like `KUBE_COVERAGE_FLUSH_INTERVAL` that falls back to a default now than have to make it tuneable later.